### ISSUE TITLE
better support for "scopes"

### ISSUE THREAD

**Issue by maelle** [cynkra/fledge#345]
e.g. group and order them in `update_news()`

**Comment by rtaph** [comment]
One thing I would find useful is the ability to have scopes as top-level bullets with nested bullets under them (e.g. instead of having scopes as sub-headers).

**Comment by maelle** [comment]
in that case, why not use types? (genuine question)

**Comment by rtaph** [comment]
Yes, type-grouping as headers is great (h2 headers for fix, feat, etc.)! I think that is already in place on the main branch now and it's super helpful 🎉 . I am just finding myself manually re-organizing the h3 scope headers into bullets and nesting their child bullets.
